turtleGraphics.js

```diff
@@ -0,0 +1,107 @@
+class Turtle {
+    constructor (x, y) {
+        this.x = x;
+        this.y = y;
+        this.direction = "east";
```

Comment:
In this case, the "east" is something we call "magic string" what this means, is that you are using it in many places throughout your class, so I would add constants at the very top of the page with these variables like so:

```
const NORTH_DIRECTION = 'north';
const SOUTH_DIRECTION = 'south';
const EAST_DIRECTION = 'east';
const WEST_DIRECTION = 'west';
```

then use it throughout your application.

```
this.direction = EAST_DIRECTION;
```

The advantages of this approach is as follows:

* Easier to maintain and spread throughout your app.
* You are decoupling the value from the instances they are used.
* If you wanted to export the directions to other  modules,  you'd be able to reuse the same variables.